RNA-seq/00b-additional_approaches.md

```diff
@@ -44,34 +47,31 @@ Run the following command:
 
 ```bash
 multiqc \
-  QC/gastric_cancer \
-  data/quant/gastric_cancer \
-  --outdir QC/gastric_cancer/
+  ~/shared-data/training-modules/RNA-seq/QC/gastric-cancer \
+  data/gastric_cancer/salmon_quant/ \
+  --outdir QC/gastric_cancer
 ```
 
 Once this completes, you should have a report at `QC/gastric_cancer/multiqc_report.html`.
 
 ## Decoy sequence-aware selective alignment with Salmon
 
-In mid-2019, the folks that develop Salmon posted a preprint called _Alignment and mapping methodology influence transcript abundance estimation_ ([Srivastava et al. _bioRxiv._ 2019.](https://doi.org/10.1101/657874)).
-We're summarizing our understanding of the work here, but we encourage you to take a look at the preprint yourself.
-
-Srivastava et al. demonstrate that transcript abundance estimates from lightweight mapping approaches like Salmon can differ from abundance estimates derived from traditional alignment approaches in experimental data.
+The folks that created Salmon demonstrated that transcript abundance estimates from lightweight mapping approaches like Salmon can differ from abundance estimates derived from traditional alignment approaches in experimental data ([Srivastava _et al._ 2019.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02151-8)).
 This differs from the takeaway of most prior work comparing lightweight mapping and traditional alignment; this is very likely due to the typical focus on _simulated data_ rather than experimental data.
 
-In the preprint, the authors introduced a new approach termed "selective alignment" that is less computationally costly than traditional alignment while still offering improvements over lightweight mapping. 
+The authors introduced a new approach termed "selective alignment" that is less computationally costly than traditional alignment while still offering improvements over lightweight mapping.
 
-The current version of Salmon (as of writing this) `v1.2.0` allows users to input sequences from unannotated genomic loci that are similar to annotated transcripts, termed decoy sequences, to avoid falsely mapping fragments that arise from these unannotated regions to transcripts.
-This is termed a `salmon_partial_sa_index` [here](https://github.com/COMBINE-lab/salmon/tree/91091fc3650a3220f657a9f31616916513f0ad02#pre-computed-decoy-transcriptomes). 
+The current version of Salmon (as of writing this) allows users to input sequences from unannotated genomic loci that are similar to annotated transcripts, termed decoy sequences, to avoid falsely mapping fragments that arise from these unannotated regions to transcripts.
+This is termed a `salmon_partial_sa_index` [here](https://github.com/COMBINE-lab/salmon/tree/91091fc3650a3220f657a9f31616916513f0ad02#pre-computed-decoy-transcriptomes).
 As of `v1.0.0`, you can use the _full genome_ as decoy ([ref](https://github.com/COMBINE-lab/salmon/tree/91091fc3650a3220f657a9f31616916513f0ad02#pre-computed-decoy-transcriptomes)).
 
```

Comment:
```suggestion
To apply the selective alignment method, you will need to first have an index that includes not only the transcripts of interest, but also a set of other potentially mapped sequences. 
Instructions on creating such an index can be found in [the Salmon documentation](https://salmon.readthedocs.io/en/latest/salmon.html#preparing-transcriptome-indices-mapping-based-mode).
One option is to include "decoy sequences" in Salmon index from genomic loci that are similar to annotated transcripts, to avoid falsely mapping fragments that arise from these unannotated regions to the transcripts of interest.
Alternatively (recommended), you can use the [_full genome_ as a decoy](https://combine-lab.github.io/alevin-tutorial/2019/selective-alignment/)
```